doc: add changelog entry for PMIC samples (#24178)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

PMIC samples
------------

* Updated:

>   * The nPM1300: Fuel Gauge sample has been renamed to :ref:`npm13xx_fuel_gauge`.

Reviewer: ```suggestion
  * By renaming the nPM1300: Fuel Gauge sample to :ref:`npm13xx_fuel_gauge`.
```

Reviewer: This reads better with "Updated", doesn't it?

Reviewer: Understood. We have a bit different approach in the docs. We have the "headings" Added, Updated, Fixed and Removed. And the text coming after should read well with those.

    The wiring is changed for all targets.
    Refer to the :ref:`Wiring table <npm13xx_fuel_gauge_wiring>` in the sample documentation for details.

>   * The nPM1300: One button sample has been renamed to :ref:`npm13xx_one_button`:

Reviewer: ```suggestion
  * By renaming the nPM1300: One button sample to :ref:`npm13xx_one_button`:
```
